Common/ArtificerItems.cs

```diff
@@ -104,18 +106,18 @@ public override void ModifyTooltips(Item item, List<TooltipLine> tooltips)
             {
                 coolTimeColorHex = "[c/78be78:";
                 tooltips.Insert(index, new TooltipLine(Mod, "cooldownTime",
-                "Cooldown time: " + $"{coolTimeColorHex}{finalCooldown} seconds] (" + $"Base: {baseCooldown} sec)"));
+                Language.GetTextValue("Mods.ArtificerMod.CommonItemtooltip.Cooldown.CooldowntimeColored").FormatWith(baseCooldown, finalCooldown)));
             }
             else if (negativeMods)
             {
                 coolTimeColorHex = "[c/be7878:";
                 tooltips.Insert(index, new TooltipLine(Mod, "cooldownTime",
-                "Cooldown time: " + $"{coolTimeColorHex}{finalCooldown} seconds] (" + $"Base: {baseCooldown} sec)"));
+                Language.GetTextValue("Mods.ArtificerMod.CommonItemtooltip.Cooldown.CooldowntimeColored").FormatWith(baseCooldown, finalCooldown)));
             }
             else
             {
                 tooltips.Insert(index, new TooltipLine(Mod, "cooldownTime",
-                "Cooldown time: " + $"{finalCooldown} seconds (" + $"Base: {baseCooldown} sec)"));
+                Language.GetTextValue("Mods.ArtificerMod.CommonItemtooltip.Cooldown.Cooldowntime").FormatWith(baseCooldown, finalCooldown)));

```

Comment:
The tooltip for 'positive' cooldown times will need its own `CommonItemtooltip`, as it is colored differently from the negative tooltip;

- `[c/78be78:` is correct for the negative tooltips; as this colors the time text red.
- For the positive tooltip, we want `[c/be7878:`, which will color the time text green.

All you need to do is add an appropriate `CommonItemtooltip` to `localization` and use it here